sickbeard/webserve.py

```diff
@@ -1224,6 +1224,25 @@ def getDBcompare():
             logger.log(u"Checkout branch couldn't compare DB version.", logger.ERROR)
             return json.dumps({"status": "error", 'message': 'General exception'})
 
+    def getSeasonSceneExceptions(self, indexer, indexer_id):  # @TODO: OMG, You can use this as an example to create the Api for the season exception indicators
+        """Get show name scene exceptions per season
+
+        :param indexer: The shows indexer
+        :param indexer_id: The shows indexer_id
+        :return: A json with the scene exceptions per season.
+        """
+
+        exceptionsList = {}
+        exceptionsList['seasonExceptions'] = sickbeard.scene_exceptions.get_all_scene_exceptions(indexer_id)
+
+        xem_numbering_season = {}
+        for tvdb_season_ep, anidb_season_ep in get_xem_numbering_for_show(indexer_id, indexer).iteritems():
+            if not xem_numbering_season.get(str(tvdb_season_ep[0])):
+                xem_numbering_season[str(tvdb_season_ep[0])] = str(anidb_season_ep[0])
+
```

Comment:
@medariox and even better yet, make it a dict comprehension:
```python
xem_numbering_season = {
    season[0]: xem[0] 
    for season, xem in get_xem_numbering_for_show(indexer_id, indexer).items()
}
```
also exceptionLists should be both snake_case (and keys shouldn't be camel case) and a dict literal  e.g.:
```python
exception_lists = {
    'season_exceptions': sickbeard.scene_exceptions.get_all_scene_exceptions(indexer_id),
    'xem_numbering': xem_numbering_season,  # can also use dict comprehension here directly.
}